src/utilities/convertRemToPx.ts

```diff
@@ -0,0 +1,4 @@
+export const convertRemToPx = (value: string) => {
+  const unitRem = parseFloat(value);
+  return unitRem / 0.0625;
```

Comment:
could you make it a bit more explicit what `0.0625` is? Something like `defaultRootFontSize`